Sources/SwiftFormatPrettyPrint/TokenStreamCreator.swift

```diff
@@ -47,6 +47,10 @@ private final class TokenStreamCreator: SyntaxVisitor {
   /// stream.
   private var pendingMultilineStringSegmentPrefixLengths = [TokenSyntax: Int]()
 
+  /// Lists tokens that shouldn't be appended to the token stream as `syntax` tokens. They will be
+  /// printed conditionally using a different type of token.
+  private var ignoredTokens =  [TokenSyntax]()
```

Comment:
In practice this is probably minor, but since `TokenSyntax`es are `Hashable`, you can use a `Set<>` instead of an array here to make the lookup closer to constant-time.

Reply:
Done.